src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -497,8 +499,12 @@ void medVtkViewItkDataImageInteractor::setMaxIntensity(double max)
 
 QWidget* medVtkViewItkDataImageInteractor::buildToolBarWidget()
 {
+    d->toolbar = new QWidget();
```

Comment:
It seems that it's the same behavior in buildToolBoxWidget (the creation of the QWidget), and also it's stored as QPointer. You'll tell me what you prefer

Reply:
Also it seems the same behavior in vtkDataMeshInteractor::buildToolBarWidget()

Reply:
I know it's unfortunately the same ugly behavior in some other part in the code. But, we will must change it in a near future. Then we don't need to duplicate this bad way to avoid future painfull work.

It's absolutly not your fault. I perfectly understand you just follow the "classic" style.

Reply:
I change the code myself it's now ok to merge